app/views/works_mailer/reject_email.html.erb

```diff
@@ -3,7 +3,7 @@
 <p>Your deposit, “<%= @work.title %>” to the <%= @work.collection_name %>
   collection in the Stanford Digital Repository, has been returned to you with the following comments:</p>
 
-<p><%= @work.last_rejection_description %></p>
+<p style="margin-left: 30px; font-weight: bold;"><%= @work.last_rejection_description %></p>
```

Comment:
Can you use relative units (e.g. `rem`) rather than `px` for better accessibility?

Reply:
Fixed.